armi/bookkeeping/db/database3.py

```diff
@@ -741,9 +727,11 @@ def load(
         )
         root = comps[0][0]
 
-        # ensure the max assembly number is correct, unless the user says no
         if updateGlobalAssemNum:
-            updateGlobalAssemblyNum(root)
+            runLog.warning(
+                "The method input `updateGlobalAssemNum` is no longer used.",
+                single=True,
+            )
```

Comment:
could also be a candidate for [DeprecationWarning](https://docs.python.org/3/library/exceptions.html#DeprecationWarning)